index.html

```diff
@@ -290,7 +294,21 @@ <h3>MathML Element Mappings</h3>
 		  </td>
 		  <td class="axapi">
 		    <span class="role">AXRole: <code>NSAccessibilityGroupRole</code></span><br />
-		    <span class="subrole">AXSubrole: <code>AXMathMultiscript</code></span>
+		    <span class="subrole">AXSubrole: <code>AXMathMultiscript</code></span><br />
+		    <span class="attrs">AXAttributes:<ul>
+		      <li><code>AXMathPostscripts</code> is an array of dictionaries of
+		      <code>AXMathSubscript</code> and <code>AXMathSupscript</code> pointing to
+		      postsubscript and postsupscript elements, i.e. N and N + 1 in-flow children
+		      starting from a second in-flow child and preceeding
+		      <a data-cite="MathML3/chapter3.html#presm.mprescripts">`mprescripts`</a>
+		      element;
```

Comment:
Note that `<mprescripts>` is optional.  I added some definitions for children, if that's easier to refer to:
https://mathml-refresh.github.io/mathml-core/#dfn-multiscripts-base
https://mathml-refresh.github.io/mathml-core/#dfn-multiscripts-postscripts
https://mathml-refresh.github.io/mathml-core/#dfn-subscript-superscript-pair
https://mathml-refresh.github.io/mathml-core/#dfn-mmultiscripts-prescripts

Reply:
this is a great idea! however my understanding we can't refer yet to https://mathml-refresh.github.io, @joanmarie is this correct?

I adjusted wording to make mprescripts optional